src/components/30-organisms/table-sortable/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/table-sortable",
+  "version": "0.0.0",
+  "description": "> TODO: description",
```

Comment:
I vote for writing something here. Just a little explanation, no rocket science stuff. Everything is better than a todo imho.

Reply:
agree

Reply:
done